### ISSUE TITLE
Mouse stuck in corner only on Wayland

### ISSUE THREAD

**Issue by JakubFojtik** [home-sweet-gnome/dash-to-panel#1805]
**Describe the bug**
Mouse gets stuck in the Applications corner and can't move in the direction of the panel. Has to return horizontally a bit, then it can move freely again.
Tried on all panel positions. Only happens on Wayland even after re-log.

**Linux distribution and version**
Fedora 37

**GNOME Shell version**
43.2

**Dash-to-Panel version**
52

**Where was Dash-to-Panel installed from?**
The GNOME extensions website

**Screenshots / Video captures**
Trying to move the mouse downwards when in the corner, it can only move horizontally up to a small distance, then the invisible barrier ends:

![mouse_cant_move_down(1)](https://user-images.githubusercontent.com/12374584/207165462-9536e72b-53ed-4302-bf06-c2eae537a0b3.gif)

**Comment by charlesg99** [comment]
What screen configuration do you have? Is this a new issue? If so when did you first notice it?

**Comment by JakubFojtik** [comment]
1920x1080x144, single monitor. No custom DM/Compositors/anything. NVIDIA card/driver. Don't know what else there is to screen configuration.
Noticed after upgrade to Fedora 37, but that's probably also the first time I ran this extension on Wayland, sorry.

**Comment by charlesg99** [comment]
Thanks! Could you try the master branch?

**Comment by charlesg99** [comment]
Thanks! Made another adjustment on the master branch, can you pull/install again? Seems to work correctly now.

**Comment by JakubFojtik** [comment]
Sadly, I don't see much difference. Moving left from the corner still blocks moving down.
I did git pull, make install, disable old ext, relog, enable new. Is something else needed for a proper reload of changes?

![Screencast from 2022-12-20 13-23-54](https://user-images.githubusercontent.com/12374584/208667214-68f23333-4bd4-4d52-a5d0-54b4ced51f16.gif)

**Comment by charlesg99** [comment]
Sounds like you did it right, maybe my tests are bad as my wayland machine is still on gnome-shell 42.

**Comment by charlesg99** [comment]
Is it fixed on later gnome-shell versions?

**Comment by JakubFojtik** [comment]
> Is it fixed on later gnome-shell versions?

Nope, at least not on GNOME Shell 44.10 and Dash-to-Panel version 56.

**Comment by JakubFojtik** [comment]
Still there on Gnome 47 DtP 64. Any chance someone could reproduce it?
